src/more/more.rs

```diff
@@ -137,30 +179,74 @@ fn reset_term(term: &mut redox_termios::Termios) {
 fn more(matches: getopts::Matches) {
     let files = matches.free;
     let mut f = File::open(files.first().unwrap()).unwrap();
```

Comment:
If you have time, it would be great if you could remove all these `unwrap()`s.  This function should return `Result<(), SomeErrorType>`, which we then check in `uumain()`.